claims_test.go

```diff
@@ -0,0 +1,64 @@
+package jwt_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
+	"github.com/dgrijalva/jwt-go/test"
+)
+
+const (
+	expireAtInt = 100
+	nowInt = 200
+)
+
+var claimsTestData = []struct {
+	name   string
+	need   string
+	claims jwt.Claims
+}{
+	{
+		name:   "Map claims expired json.Number",
+		need:   "Given the need to test validating a MapClaims past its json.Number expire-at time.",
```

Comment:
Why not use the constant here?  String comparing of errors is generally ill advised.